src/features/home/DashboardPanelLeft.tsx

```diff
@@ -1,42 +1,58 @@
 import './dashboardPanelLeft.less';
 import { PlusCircleOutlined } from '@ant-design/icons';
 import { Button, Col, Divider, Row, Typography } from 'antd';
-import React from 'react';
 import { useNavigate } from 'react-router-dom';
 
 import type { Market } from '../../api-spec/generated/js/types_pb';
+import type { RootState } from '../../app/store';
+import { useTypedSelector } from '../../app/store';
 import { CREATE_MARKET_ROUTE } from '../../routes/constants';
 import type { LbtcUnit } from '../../utils';
-import { fromSatsToUnitOrFractional } from '../../utils';
+import { LBTC_ASSET } from '../../utils';
 import { useListMarketsQuery, useTotalCollectedSwapFeesQuery } from '../operator/operator.api';
+import type { PriceFeedQueryResult } from '../rates.api';
+import { convertAssetToCurrency } from '../rates.api';
 
 const { Title } = Typography;
 
 interface DashboardPanelLeftProps {
   lbtcUnit: LbtcUnit;
+  priceFeed: PriceFeedQueryResult;
 }
 
-export const DashboardPanelLeft = ({ lbtcUnit }: DashboardPanelLeftProps): JSX.Element => {
+export const DashboardPanelLeft = ({ lbtcUnit, priceFeed }: DashboardPanelLeftProps): JSX.Element => {
   const navigate = useNavigate();
+  const { currency, network } = useTypedSelector(({ settings }: RootState) => settings);
   const { data: listMarkets } = useListMarketsQuery();
+  const { data: prices, isLoading: isLoadingPrices, isError: isErrorPrices } = priceFeed;
   const activeMarkets = listMarkets?.marketsList.filter((m) => m.tradable).length || 0;
   const pausedMarkets = (listMarkets?.marketsList.length ?? 0) - activeMarkets;
   //
   const markets = listMarkets?.marketsList.map((m) => m.market as Market.AsObject);
   const { data: totalCollectedSwapFees } = useTotalCollectedSwapFeesQuery(markets);
 
+  const TotalCollectedSwapFees =
+    totalCollectedSwapFees &&
```

Comment:
I would allow `convertAssetToCurrency` to take `undefined` as amount. In which case the helper should return `undefined`. 
```
  const totalCollectedSwapFeesAsPreferredCurrency = convertAssetToCurrency({
    asset: LBTC_ASSET[network],
    amount: totalCollectedSwapFees,
    network: network,
    preferredCurrency: currency,
    preferredLbtcUnit: lbtcUnit,
    prices: prices,
    useSymbol: true,
  });
```
Then we use a ternary operator to display `N/A` in case of issue.
```
  {!isLoadingPrices && !isErrorPrices && totalCollectedSwapFeesAsPreferredCurrency
            ? totalCollectedSwapFeesAsPreferredCurrency
            : 'N/A'}
```